single_chart_migration/migration.sh

```diff
@@ -159,11 +159,64 @@ find_releases() {
     fi
 }
 
+mongo_trail_migration() {
+
+    namespace=$1
+    mongodeployed=false
+    echo "mongotrailmigration case"
+    # Check for the presence of required files.
+    if [ ! -f "mongo.yaml" ]; then
+        echo "mongo.yaml is missing. Please download it. (curl https://raw.githubusercontent.com/portworx/helm/master/single_chart_migration/mongo.yaml -o mongo.yaml)"
+    fi
+    if [ ! -f "migrationctl" ]; then
+        echo "migrationctl is missing. Please download it. (curl https://raw.githubusercontent.com/portworx/helm/master/single_chart_migration/migrationctl -o migrationctl)"
+    fi
+    # Get the storage class from the etcd PVC and use it for mongo PVC.
+    sc=`$kubectl_cmd get pvc pxc-etcd-data-pxc-backup-etcd-0 -n $namespace -o yaml | grep storageClassName | awk '{print $2}'`
+    echo "using storage class for mongo $sc"
+    # update the storageclass in mongo.yaml
+    sed -i 's|'storageClassName:.*'|'"storageClassName: $sc"'|g' mongo.yaml
+    # Install mongoDB
+    $kubectl_cmd apply -f ./mongo.yaml -n $namespace
+    echo "Waiting for mongoDB to be in running state"
+    for i in $(seq 1 100) ; do
+        replicas=$(kubectl get statefulset -n px-backup pxc-backup-mongodb-trail -o json | jq ".status.readyReplicas")
```

Comment:
We should avoid using jq as it may not be be present in some machines.

Reply:
Also use $kubectl_cmd instead of kubectl also namespace is hardcoded here.

Reply:
Removed Jp and fixed the namespace and kubectl issues.